agentex/templates/admin/agentex/allcalibrators.html-e

```diff
@@ -10,7 +10,7 @@
 
 {% block breadcrumbs %}
 <div class="breadcrumbs">
-<a href="{% url admin:index %}">Home</a> > <a href="{% url admin:agentex_event_changelist %}">Planets</a>
+<a href="{% url admin:index %}">Home</a> > <a href="{% url 'agentex_event_changelist' %}">Planets</a>
```

Comment:
This is not the correct way for making admin URLs. See [this part of the docs](https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#admin-reverse-urls)